modules/services/window-managers/i3-sway/lib/options.nix

```
@@ -819,4 +819,12 @@ in {
     description = "Default launcher to use.";
     example = "bemenu-run";
   };
+
+  defaultWorkspace = mkOption {
+    type = types.nullOr types.str;
+    default = null;
+    description =
+      "Has to be the value of the keybinding (of the default Workspace).";
```

Sorry for another comment: I think this description should be reworded. Maybe:

```suggestion
    description = ''
      The default workspace to show when ${if isSway "sway" else "i3"} is launched.
      This must to correspond to the value of the keybinding of the default workspace.
    '';
```

---

Done :)